CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to c2cgeoportal
+
+Thanks for your interest in contributing to a GeoMapFish project.
+
+## Asking Questions
+
+Please ask questions about using the library on [Stack Overflow using the tag 'openlayers']
+(http://stackoverflow.com/questions/tagged/openlayers).
+
+When you want to get involved and discuss new features or changes, please use [the mailing list]
+(https://groups.google.com/forum/#!forum/openlayers-dev).
+
+
+## Submitting Bug Reports
+
+Please use the [GitHub issue tracker](https://github.com/camptocamp/c2cgeoportal/issues). Before creating a
+new issue, do a quick search to see if the problem has been reported already.
```

Comment:
Greate we are using github issues and not JIRA.

Reply:
To be discussed, @ybolognini